### ISSUE TITLE
[Bug] [KDA] nan in chunk_kda_fwd_kernel_intra_sub_inter when num_v_heads != num_heads

### ISSUE THREAD

**Issue by polarispw** [fla-org/flash-linear-attention#624]
### Checklist

- [x] I have checked [FAQs](https://github.com/fla-org/flash-linear-attention/blob/main/FAQs.md) and existing issues for similar problems
- [x] Please report this bug in English to ensure wider understanding and support

### Describe the Bug

Encountered triton error when setting num_v_heads == 2 * num_heads.
Seems that the last dim of g and beta mismatch other inputs like q,k

https://github.com/fla-org/flash-linear-attention/blob/a011ffd6b627dc749bed9198fc1c379de58aa6ac/fla/layers/kda.py#L221

The following fix may help? :)

```
q, k, g = (repeat(x, '... h d -> ... (h g) d', g=self.num_v_heads // self.num_heads) for x in (q, k, g))
beta = repeat(beta, '... h -> ... (h g)', g=self.num_v_heads // self.num_heads)
```

### Steps to Reproduce the Bug

```
import torch
from fla.layers import KimiDeltaAttention

bs, num_heads, num_v_heads, seq_len, hidden_size = 4, 16, 32, 256, 1536
kda = KimiDeltaAttention(hidden_size=hidden_size, num_heads=num_heads, num_v_heads=num_v_heads, head_dim=128, expand_v=1.0, mode='chunk').to(dtype=torch.bfloat16, device='cuda')
print(kda)

x = torch.randn(bs, seq_len, hidden_size).bfloat16().cuda()
print(x)
y, _, _ = kda(x)
print(y)

loss = y.norm(p=2)
print(loss)
```

### Expected Behavior

not any 'nan' after forwarding

### Environment Information

1. Torch:2.5.1
2. Triton:3.1

**Comment by zhiyuan1i** [comment]
At the moment, we assume that the q, k, and v heads are equal, with support for other scenarios to be added in the future.

**Comment by sustcsonglin** [comment]
thanks for reporting. actually i think it is a good workaround. fixed
